lib/orderbook/OrderBook.ts

```diff
@@ -257,10 +258,14 @@ class OrderBook extends EventEmitter {
     }
   }
 
-  private isOwnOrder = (type: OrdersMap) => {
+  private isOwnOrdersMap = (type: OrdersMap) => {
     return type === this.ownOrders;
   }
 
+  private isOwnOrder = (order: orders.StampedOrder) => {
```

Comment:
I think this doesn't need to be an instance method - it can just be moved out of the class - and made a regular method, and you can tell the compiler that this checks for `StampedOwnOrder` types. I also think it would be nice to check for `localId` as well and for the right types:

```javascript
function isOwnOrder(order: orders.StampedOrder): order is StampedOwnOrder {
  return order['hostId'] === undefined && typeof order['localId'] === 'string';
}
```

Reply:
Right, I made it static instead because I'm usually not in favor of making function hanging in the file, specially if they do relate to the class.

I added the `localId` check, we'll need to keep an eye on it in case it will become optional.

Reply:
Did you commit those changes yet? I don't see them.

I think if you want to keep it as part of the class and make it static and add the `order is orders.StampedOwnOrder` to it that'd be fine. It actually might make more sense to move this function to the orders.ts file where `orders.StampedOwnOrder` is defined.

Reply:
Just noticed that the push failed :\